### ISSUE TITLE
Filament A60 not recognized as a Hue bulb, but Unknown Zigbee device

### ISSUE THREAD

**Issue by JelleYpeZn** [sebbebebbe/com.philips.hue.zigbee#379]
Recently I bought a Hue Filament A60 bulb (LTA005) but after pairing I got the message 'Unknown Zigbee device'. It is possible to put the device on/off, dim and adjust the color temperature. But I cannot design a flow with this device in it, because it is not listed in de Hue without a bridge app. How to solve this problem?

**Comment by JohanBendz** [comment]
Hi @JelleYpeZn, can you please add it as a generic Zigbee device, go to developer tools and make an interview and post it here?
https://tools.developer.homey.app/tools/zigbee

**Comment by JelleYpeZn** [comment]
Hello Johan,

Hereby I send you 2 screenpictures, taken after I pressed the button Interview.
During the process the bulb goes on and off.
The third screenpicture shows the list of devices shown when select ‘Philips Hue, without the bridge’ in the list of apps (while creating a flow).

Thank you in advance for your help.

Greetings,
Jelle Dijkstra

**Comment by JelleYpeZn** [comment]
Good morning Johan,

I was reading your mail again and now I have a question about the remark: “add it as a generic Zigbee device”. Do you mean that I have to remove the device and pair it again with the Homey, but then as a generic device? And if so, how can I do that, with which app?

Greetings,
Jelle Dijkstra

**Comment by Thorarin** [comment]
I've also bought this particular light bulb today (LTA005).
Here is the interview result: [LTA005 interview.txt](https://github.com/JohanBendz/com.philips.hue.zigbee/files/8838082/LTA005.interview.txt)

I could try to make a PR, but I mostly have some experience with Z-Wave drivers, not so much Zigbee. I'm not sure if it can be lobbed together with any of the existing drivers or if it should be separate. I think it would have to be separate from the existing Filament A60 (LWA004), because it has a color temperature capability.

Additional properties of this light bulb can be found here:
[philips-hue.com product page](https://www.philips-hue.com/nl-nl/p/hue-white-ambiance-filament-1-pack-a60-e27-standard-met-zichtbare-gloeidraad/8719514301429#overview)

TLDR:
Brightness: 550 lm (at 2700K)
Max power draw: 7 W
Standby power draw: 0.5 W max
Color temperature 2200K - 4500K

**Comment by JohanBendz** [comment]
Added to next release. Sorry for the delay.
